Machine Learning/ProblemChild/job_configs/experimental-rare-process-by-user-problemchild.txt

```diff
@@ -0,0 +1,25 @@
+PUT _ml/anomaly_detectors/experimental-rare-process-by-user-problemchild
```

Comment:
Remove `PUT` statements from `json` files.

change file extension from `txt` to `json`